.github/workflows/verify-db-migration.yml

```
@@ -0,0 +1,267 @@
+            --startup-project ./src/Altinn.Correspondence.API
+     
+      - name: Create directory to save created IDs
+        run: |
+          mkdir -p ./Test/Altinn.Correspondence.Tests/test-results
+
+
+      # ============================================
+      # STEP 2: Test old code with old database
+      # ============================================
+      - name: Start old version API and run test
+        uses: grafana/k6-action@e4714b734f2b0afaabeb7b4a69142745548ab9ec # v0.3.1
```

actionlint (1.7.10)

[error] unexpected key "uses" for step to run shell command. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "working-directory"

(syntax-check)